test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -282,55 +295,134 @@ public void testGenome() throws Exception {
         Assert.assertEquals("1", "" + assemblyIndex.getKeyProperties().get("contigs"));
         System.out.println("*** end testGenome***");
     }
+
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
I realize this way is easier but we really should check that the exception message is correct

Reply:
Done.